src/fu-util-common.c

```diff
@@ -1217,6 +1217,10 @@ fu_util_device_flag_to_string(guint64 device_flag)
 		/* TRANSLATORS: we might ask the user the recovery key when next booting Windows */
 		return _("Full disk encryption secrets may be invalidated when updating");
 	}
+	if (device_flag == FWUPD_DEVICE_FLAG_NO_AUTO_REMOVE) {
```

Comment:
once you move the flag to an internal flag the changes for this file should go away